src/analysis/network/qgsstrategy.h

```diff
@@ -23,35 +23,39 @@
 
 /**
  * \ingroup analysis
- * \class QgsArcProperter
- * \brief QgsArcProperter is a strategy pattern.
- * You can use it for customize arc property. For example look at QgsDistanceArcProperter and QgsSpeedArcProperter
+ * \class QgsStrategy
+ * \brief QgsStrategy defines strategy used for calculation of the edge cost. For example it can
+ * take into account travel distance, amount of time or money. Currently there are two strategies
+ * implemented in the analysis library: QgsDistanceStrategy and QgsSpeedStrategy.
+ * QgsStrategy implemented with strategy design pattern.
  */
-class ANALYSIS_EXPORT QgsArcProperter
+class ANALYSIS_EXPORT QgsStrategy
```

Comment:
Will fix it. Does it makes sense to change also names of the subclasses, e.g. to `QgsNetworkDistanceStrategy`?